### PR TITLE
remove old restore code

### PR DESCRIPTION
**Pull request by snopoke**
@dannyroberts @czue

### CODE REVIEW

File: corehq/ex-submodules/casexml/apps/phone/caselogic.py [hunk 1]
```diff
@@ -349,6 +246,18 @@ def batches(self):
                 case_sharing=self.case_sharing
             )
 
+    def get_all_case_updates(self):
+        """
+        Returns a generator that yields the case updates for this user.
+        Iterating through the updates also has the effect of updating this object's GlobalSyncState.
+        """
+        def get_updates(batch):
+            logger.debug(batch)
```
Comment (czue) [inline corehq/ex-submodules/casexml/apps/phone/caselogic.py]: realize it just moved, but do you intend to keep this here?

**Comment by czue** [comment]
great addition/deletion ratio!

**Comment by czue** [comment]
:+1: from me when tests pass

**Comment by dannyroberts** [comment]
:birthday: It's a good day after all :tada:

File: corehq/ex-submodules/casexml/apps/phone/caselogic.py (hunk 1 above)
Comment (snopoke) [inline corehq/ex-submodules/casexml/apps/phone/caselogic.py]: It is useful if you're debugging especially as it includes the couch keys.

**Pull request merged**